sections/section_util.go

```diff
@@ -0,0 +1,62 @@
+package sections
```

Comment:
This is unused right now, but was an idea to try to simplify the parsing of the segments